code/modules/antagonists/thief/thief.dm

```diff
@@ -13,11 +13,23 @@
 	var/list/thief_flavor
 	///if added by an admin, they can choose a thief flavor
 	var/admin_choice_flavor
+	///an area marked as the thieves guild- makes thieves happier to be in, and all thieves of the round know of it. only has a 20% chance of existing in a round.
+	var/static/area/thieves_guild
+	///bool checked for the first thief in a round to decide if there should be one this round
+	var/static/decided_on_guild = FALSE
 
 /datum/antagonist/thief/on_gain()
+	if(!decided_on_guild)
+		decided_on_guild = TRUE
+		if(prob(20))
```

Comment:
I didn't because thieves roll in packs. 5 Thieves roundstart at a 20% chance gives rounds a 2/3rd chance to have a thieves guild just from roundstart. I think it's a lot better to just have this be static, because while I could make this a very low probability and tweak it to the average amount of thieves, one I don't have that data, two that's a lot of work for very little accomplished and three it kills the variety of having large guilds and small guilds since small guilds will near never exist and large ones will always exist turning thieves into more of a team antag than they should be

Reply:
no? it gives rounds a 1/5th chance for a thieves guild, that prob(20) only runs once because it's static

Reply:
It doesn't roll again. It's static, it's going to roll the 20% chance once until the server restarts.